hack/.notableofcontents

```diff
@@ -31,3 +31,4 @@ docs/ovs-offload.md
 docs/prometheus-integration.md
 docs/security.md
 docs/windows.md
+docs/network-requirements.md
```

Comment:
The list in alphabetical order, could you make it follow the convention?

Reply:
sure, just updated it.